mineinabyss-features/src/main/kotlin/com/mineinabyss/helpers/CosmeticHelpers.kt

```diff
@@ -1,41 +1,33 @@
 package com.mineinabyss.helpers
 
-import com.mineinabyss.components.cosmetics.cosmetics
-import com.mineinabyss.components.playerData
+import com.mineinabyss.components.cosmetics.Cosmetics
+import com.mineinabyss.geary.papermc.access.toGeary
 import com.mineinabyss.mineinabyss.core.mcCosmetics
-import io.lumine.cosmetics.api.cosmetics.ItemCosmetic
-import io.lumine.cosmetics.managers.back.BackAccessory
+import io.github.fisher2911.hmccosmetics.api.CosmeticItem
+import io.github.fisher2911.hmccosmetics.api.HMCCosmeticsAPI
+import io.github.fisher2911.hmccosmetics.gui.ArmorItem
 import io.lumine.cosmetics.managers.gestures.GestureManager
-import io.lumine.cosmetics.managers.hats.Hat
 import io.lumine.cosmetics.players.Profile
-import org.bukkit.Material
 import org.bukkit.entity.Player
-import org.bukkit.inventory.ItemStack
 
 fun Player.playGesture(gesture: String) {
-    mcCosmetics.profiles.getProfile(name).ifPresent { profile: Profile? ->
-        mcCosmetics.gestureManager.getCosmetic(gesture).ifPresent { gesture ->
-            cosmetics.gesture = gesture
+    mcCosmetics.profiles.getProfile(name).ifPresent profile@{ profile: Profile? ->
```

Comment:
You can probably just get or else return, perhaps get or null ?: return.

I generally dislike the extra indentation ifPresent adds